### DISCUSSION TITLE
How to receive and send video_note

### DISCUSSION THREAD

**Discussion by rodnovspb** [def-studio/telegraph#614] [Q&A]
Good day! Please tell me, is it possible to receive and send video_note?

**Comment by fabio-ivona** [comment]
Hi, not yet

feel free to open a pull request if you wish to help

**Reply by rodnovspb** [reply]
Thanks for a good product! I don't have enough knowledge to help yet, sorry

**Reply by fabio-ivona** [reply]
No problem, we'll plan this feature in one of next releases
